src/Storage/Device.php

```diff
@@ -5,10 +5,15 @@
 abstract class Device
 {
     /**
-     * Max chunk size while transfering file from one device to another
+     * Max chunk size while transferring file from one device to another
      */
     protected int $transferChunkSize = 20000000; //20 MB
 
+    /**
+     * Sets the maximum number of keys returned to the response. By default, the action returns up to 1,000 key names.
+     */
+    protected const MAX_KEYS = 1000;
```

Comment:
Let's name this `MAX_PAGE_SIZE` and set to `PHP_INT_MAX` here, then override to 1000 for S3

Reply:
Done 👍